multiterm/src/context-menu.vala

```diff
@@ -68,6 +68,20 @@ namespace MultiTerm
 			item.show();
 		}
 
+		private Gtk.MenuItem add_image_menu_item(string label_text, string icon_name)
```

Comment:
I guess this should be guarded with `#if MULTITERM_GTK3` too? IIRC Vala compiler warns about unused private functions.

Reply:
Done. Even if not it shouldn't be callable on gtk2.

Reply:
Yeah, just prevents some warnings from the Vala compiler because it would be unused.